ospsuite-r-specifics/collaboration_guide.md

```diff
@@ -0,0 +1,122 @@
+# R Development Collaboration Guide
+
+## Rules
+
+The following set of rules should be followed by all contributors and checked by
+all reviewers of the OSPS R projects.
+
+1. Each change made to the codebase should have a clear purpose and exist in an
+isolated context.  
+  1.1 Each change should be related to an issue created on the project's repository.  
+  1.2 Each change should be made in a separate branch.  
+  1.3 Each change should be proposed through a pull request.  
+  
+2. Each change or addition to the code should be documented and controlled.  
+  2.1 Each change in the code should be reflected in the documentation. If 
+  documentation is not present, then it should be created.  
```

Comment:
Some code changes might be irrelevant for the documentation, e.g. bug fixes etc.

Reply:
Indeed, shall I introduce the concept of "changes that can be seen by the user". How to name that ?

Reply:
Yes, something like this. How to name.. good question :)
"Code changes that affect the API of the package" ?

Reply:
I phrased it as: 
> Each change that affects **user experience or outputs** ...